### PR TITLE
Support new state resolver

### PR DESCRIPTION
**Pull request by codeliner**
This PR also removes the `restart: always` configuration for the snapshotter. You can define a `restart_policy` when using swarm deployment: https://docs.docker.com/compose/compose-file/#/deploy
That is the better option. Auto restart is not required for dev env and makes trouble.

### CODE REVIEW

File: service/user-write/src/Model/User.php [hunk 1]
```diff
@@ -44,8 +46,5 @@ function apply(array $state, Message ...$events): array
         }
     }
 
-    $version = $state['version'] ?? 0;
-    $state['version'] = $version + count($events);
-
     return $state;
 }
```
Comment (prolic) [inline service/user-write/src/Model/User.php]: New line at end of file missing

**Review by prolic — APPROVED** [review]

**Comment by codeliner** [comment]
@prolic I totally forgot that PR. New line was finally added ;)

**Comment by prolic** [comment]
Travis errors with: No Rakefile found (looking for: rakefile, Rakefile, rakefile.rb, Rakefile.rb)

**Comment by codeliner** [comment]
@prolic Who has activated travis for micro-do? We don't have any tests yet. Travis just don't know what to do :D

**Pull request merged**